src/main/java/module_two_1/models/enums/DisciplineNames.java

```diff
@@ -0,0 +1,16 @@
+package module_two_1.models.enums;
+
+public enum DisciplineNames {
+
+    MATHS("математика"), HISTORY("история"), CULTURE("культура"),
```

Comment:
Could you please declare enum values on a new line. Also, I suggest starting using English instead of Russian for values

Reply:
Fixed